src/Microsoft.OData.ModelBuilder/Vocabularies/VocabularyTermConfigurationExtensions.tt

```diff
@@ -11,14 +11,18 @@
 <#@ import namespace="Microsoft.OData.Edm.Vocabularies" #>
 <#@ output extension=".cs" #>
 <#
-    var model = new EdmModel().ReferencedModels.FirstOrDefault(m => m.GetNamespaceAlias("Org.OData.Capabilities.V1") != null);
-    if (model == null)
+    var models = new EdmModel().ReferencedModels.Where(m => m.DeclaredNamespaces.Contains("Org.OData.Capabilities.V1") || m.DeclaredNamespaces.Contains("Org.OData.Core.V1")).ToList();
+    if (models == null)
```

Comment:
@ElizabethOkerio evaluate if this may be a better fix https://github.com/OData/odata.net/blob/master/src/Microsoft.OData.Edm/Vocabularies/CoreVocabularyModel.cs